x/concentrated-liquidity/tick.go

```diff
@@ -306,13 +306,13 @@ func (k Keeper) GetTickLiquidityNetInDirection(ctx sdk.Context, poolId uint64, t
 		return []query.TickLiquidityNet{}, err
 	}
 
-	ctx.Logger().Debug(fmt.Sprintf("userGivenStartTick %s, boundTick %s, currentTick %s\n", userGivenStartTick, boundTick, p.GetCurrentTick()))
+	ctx.Logger().Info(fmt.Sprintf("userGivenStartTick %s, boundTick %s, currentTick %s\n", userGivenStartTick, boundTick, p.GetCurrentTick()))
```

Comment:
I personally don't think these should be info logs, as info logs are already pretty swamped. As a node operator, I would want info logs to give me pertinent information to the node's operation. If we added logs for every query, it would be pretty unusable for the avg node operator. This is just an opinion though and can be disregarded if you feel strongly :)

Reply:
(Info logs are the default setting)

Reply:
I left these in case I need to debug more if the fix doesn't work for Jon. This is to be reverted before the merge